scripts/skills/actives/rf_pummel_skill.nut

```diff
@@ -34,7 +34,7 @@ this.rf_pummel_skill <- ::inherit("scripts/skills/actives/line_breaker", {
 			id = 11,
 			type = "text",
 			icon = "ui/icons/special.png",
-			text = "If the attack is successful, the target is pushed back a tile and you move into their position"
+			text = ::Reforged.Mod.Tooltips.parseString("If the attack is successful, use [Line Breaker|Skill+line_breaker] on the target")
```

Comment:
How a bout a `for free` at the end?
`... on the target for free")`

Reply:
I'll change it to `If the attack is successful, automatically use [Line Breaker|Skill+line_breaker] for free on the target`.